vagrant/config/versions.yaml

```diff
@@ -90,7 +90,7 @@ installers:
   - foreman: 'nightly'
     katello: 'nightly'
     candlepin: '4.3'
```

Comment:
@ehelms you want to also change this to nightly?

Reply:
I do not believe we arrived at a decision on the workflow, I think we were leaning towards yes but we needed to document it first.